Dockerfile

```diff
@@ -18,17 +24,17 @@ ENV AUTO_UPDATE_INTERVAL=0 \
     PATH=/opt/app-root/src/bin:/opt/app-root/bin:$PATH \
     PLATFORM="el8"
 
-RUN sed -i s/enabled=./enabled=0/g /etc/yum/pluginconf.d/subscription-manager.conf
+RUN microdnf update
 
-RUN dnf install -y 'dnf-command(config-manager)'
+RUN microdnf install -y 'yum-utils'
 
-RUN yum config-manager --add-repo http://packages.dev.3sca.net/dev_packages_3sca_net.repo
+RUN yum-config-manager --add-repo http://packages.dev.3sca.net/dev_packages_3sca_net.repo
```

Comment:
@dagbay-rh 
I don't know, I just repeat the same instruction adapted for the ubi8-minimal image.
Feel free to change it if it could be better to the apicast image.

Reply:
kk im not sure which is better honestly, we can look into that

Reply:
ohhh, wait I misread this as a completely new addition. so it was already in the apicast image, got it